### ISSUE TITLE
Brew (and maybe other) package to install easily

### ISSUE THREAD

**Issue by uded** [wagoodman/bashful#33]
Would be really cool if there would be a [brew](https://brew.sh) package (and maybe RPM and DEB) to easily install and update on development machines. This is a great replacement for Makefiles - which are rather annoying - and now it's a bit more complicated to use it...

**Comment by amir20** [comment]
This would be nice too. I recommend looking at https://goreleaser.com/ or something similar. It lets you build deb, rpm, brew for all different OS distributions. All of them are pushed to github release page.

**Comment by wagoodman** [comment]
Indeed, this was definitely needed... @amir20 I ended up going with the goreleaser approach, awesome tool! I'll give these releases a shot tomorrow (see v0.0.10 on the releases page)... I'll close this ticket if it ends up working :)

**Comment by amir20** [comment]
Nice. I like that tool too. I used it for a few projects and worked like a charm.
